bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/AbstractErrorEventDefinitionBuilder.java

```diff
@@ -41,6 +42,18 @@ public B error(final String errorCode) {
     return myself;
   }
 
+  public B errorCodeVariable(final String errorCodeVariable) {
+    final ZeebeError error = myself.getCreateSingleExtensionElement(ZeebeError.class);
+    error.setErrorCodeVariable(errorCodeVariable);
+    return myself;
+  }
+
+  public B errorMessageVariable(final String errorMessageVariable) {
+    final ZeebeError error = myself.getCreateSingleExtensionElement(ZeebeError.class);
+    error.setErrorMessageVariable(errorMessageVariable);
+    return myself;
+  }
```

Comment:
❌ We should add some JavaDoc to these methods to explain to users what they can use these methods for and what is expected of the `errorCodeVariable` and `errorMessageVariable` they provide (i.e. should be a [valid variable name](https://docs.camunda.io/docs/components/concepts/variables/#variable-names))